src/Functions/Public/iaas/Update-vRAvSphereFabricNetwork.ps1

```diff
@@ -0,0 +1,308 @@
+        [ValidateNotNullOrEmpty()]
+        [String[]]$Name,
+
+        [Parameter(Mandatory=$false,ParameterSetName="ByName")]
+        [Parameter(Mandatory=$false,ParameterSetName="ById")]
+        [String]$Ipv6Cidr,
+
+        [Parameter(Mandatory=$false,ParameterSetName="ByName")]
+        [Parameter(Mandatory=$false,ParameterSetName="ById")]
+        [bool]$IsDefault,
+
+        [Parameter(Mandatory=$false,ParameterSetName="ByName")]
+        [Parameter(Mandatory=$false,ParameterSetName="ById")]
+        [String]$Domain,
+
+        [Parameter(Mandatory=$false,ParameterSetName="ByName")]
+        [Parameter(Mandatory=$false,ParameterSetName="ById")]
+        [String]$DefaultIpv6Gateway,
+
+        [Parameter(Mandatory=$false,ParameterSetName="ByName")]
+        [Parameter(Mandatory=$false,ParameterSetName="ById")]
+        [String[]]$DnsServerAddresses,
+
+        [Parameter(Mandatory=$false,ParameterSetName="ByName")]
+        [Parameter(Mandatory=$false,ParameterSetName="ById")]
+        [bool]$IsPublic,
+
+        [Parameter(Mandatory=$false,ParameterSetName="ByName")]
+        [Parameter(Mandatory=$false,ParameterSetName="ById")]
+        [String]$Cidr,
+
+        [Parameter(Mandatory=$false,ParameterSetName="ByName")]
+        [Parameter(Mandatory=$false,ParameterSetName="ById")]
+        [String]$DefaultGateway,
+
+        [Parameter(Mandatory=$false,ParameterSetName="ByName")]
+        [Parameter(Mandatory=$false,ParameterSetName="ById")]
+        [String[]]$Tags,
+
+        [Parameter(Mandatory=$false,ParameterSetName="ByName")]
+        [Parameter(Mandatory=$false,ParameterSetName="ById")]
+        [String[]]$DnsSearchDomains,
+
+        [Parameter(Mandatory=$false)]
+        [ValidateNotNullOrEmpty()]
+        [String[]]$CloudAccountName,
+
+        [Parameter(Mandatory=$false)]
+        [ValidateNotNullOrEmpty()]
+        [String[]]$CloudAccountId
+
+
+
+    )
+    Begin {
+
+        $APIUrl = "/iaas/api/fabric-networks-vsphere"
+
+        function CalculateOutput ([PSCustomObject]$FabricNetworks) {
+            
+                [PSCustomObject]@{
+                    DefaultGateway = $FabricNetworks.defaultGateway
+                    IsPublic = $FabricNetworks.isPublic
+                    IsDefault = $FabricNetworks.isDefault
+                    ExternalRegionId = $FabricNetworks.externalRegionId
+                    CloudAccountIds = $FabricNetworks.cloudAccountIds
+                    ExternalId = $FabricNetworks.externalId
+                    Name = $FabricNetworks.name
+                    Id = $FabricNetworks.id
+                    CreatedAt = $FabricNetworks.createdAt
+                    UpdatedAt = $FabricNetworks.updatedAt
+                    OrganizationId = $FabricNetworks.organizationId
+                    OrgId = $FabricNetworks.orgId
+                    Links = $FabricNetworks._links
+                }
+            
+        }
+
+        function buildUpdateBody() {
```

Comment:
Again, I think this function should have parameters to avoid a scoping issue.

Reply:
This is updated in latest commit and tested.